programs/gateway/src/state/registrar.rs

```diff
@@ -0,0 +1,74 @@
+use crate::{
+    id,
+    tools::anchor::{DISCRIMINATOR_SIZE, PUBKEY_SIZE},
+};
+use anchor_lang::prelude::*;
+
+/// Registrar which stores Civic Pass voting configuration for the given Realm
+#[account]
+#[derive(Debug, PartialEq)]
+pub struct Registrar {
+    /// spl-governance program the Realm belongs to
+    pub governance_program_id: Pubkey,
+
+    /// Realm of the Registrar
+    pub realm: Pubkey,
+
+    /// Governing token mint the Registrar is for
+    /// It can either be the Community or the Council mint of the Realm
+    /// When the plugin is used the mint is only used as identity of the governing power (voting population)
+    /// and the actual token of the mint is not used
+    pub governing_token_mint: Pubkey,
+
+    /// The Gatekeeper Network represents the "Pass Type" that a
+    /// user must present.
+    pub gatekeeper_network: Pubkey,
+
+    /// Reserved for future upgrades
+    pub reserved: [u8; 128],
```

Comment:
With reallocs enabled on all networks, this shouldn't be needed

Reply:
We still need it until Anchor supports reallocs. It was added then removed from it and now it's in the latest version but I want to wait before we upgrade